plugin/documents.py

```diff
@@ -820,6 +820,8 @@ def render_highlights_on_main_thread() -> None:
                 if not regions:
                     continue
                 kind, multiline = tup
+                if multiline and not userprefs().show_multiline_document_highlights:
+                    continue
```

Comment:
I don't remember exactly how all of this works but shouldn't the existing same check above prevent those from being added to `kind2regions` in the first place?

Reply:
You are right, yes, thanks, it seems I had an unrelated multiline preference set and wasn't working